appng-maven-plugin/README.md

```diff
@@ -0,0 +1,81 @@
+# appNG maven plugin
+
+* replaces maven exec plugin to generate constant classes for properties and messages
+* support for eclipse m2e lifecylce
+
+## Example usage
+
+```xml
+<properties>
+	<appNG.version>1.16.2</appNG.version>
```

Comment:
Up to now, we always have been using `appNGVersion`, see e.g. https://github.com/appNG/appng-examples/blob/master/personregister/pom.xml#L12